GeneralizedMorphismsForCAP/gap/GeneralizedMorphismCategory.gi

```diff
@@ -42,8 +42,13 @@ InstallMethod( AssociatedMorphism,
                [ IsGeneralizedMorphism ],
                
   function( generalized_morphism )
+    local associated_morphism;
     
-    return DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    associated_morphism := DomainAssociatedMorphismCodomainTriple( generalized_morphism )[2];
+    
+    INSTALL_TODO_LIST_FOR_GENERALIZED_MORPHISMS( associated_morphism, generalized_morphism );
```

Comment:
Yes, sorry:

1. IsZero for generalised morphisms has for me the technical meaning, that the image is zero. In particular, no uniqueness. This is not the meaning in CAP.
2. Your second comment is exactly what I meant, I wanted the unidirectional propagation, so the use of ToDoListEntryForEqualAttributes is wrong.